### PR TITLE
Temporary work-around for P3A (if there is a hang)

### PR DESCRIPTION
**Pull request by bsclifton**
Temporary work-around for https://github.com/brave/brave-browser/issues/19280

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

**Review by SergeyZhukovsky — APPROVED** [review]
++

File: components/p3a/brave_p3a_service.cc [hunk 1]
```diff
@@ -397,6 +397,9 @@ void BraveP3AService::StartScheduledUpload() {
   }
 
   // Only upload if service is enabled.
+#if defined(OS_ANDROID)
+  p3a_enabled = false;
+#endif  // OS_ANDROID
   bool p3a_enabled = local_state_->GetBoolean(brave::kP3AEnabled);
```
Comment (rillian) [inline components/p3a/brave_p3a_service.cc:403]: This clobbers the value you've just set. Move your change down a line?

**Review by rillian — DISMISSED** [review]

File: components/p3a/brave_p3a_service.cc (hunk 1 above)
Comment (bsclifton) [reply components/p3a/brave_p3a_service.cc:403]: whoops; definitely my bad there. Thanks!

Comment (bsclifton) [reply components/p3a/brave_p3a_service.cc:400]: Fixed with https://github.com/brave/brave-core/pull/10940/commits/1333b23bdcf4ce07aa31bdf85e6a769716a5c6e3

**Comment by bsclifton** [comment]
CI passed; there was a lint warning which I'll push a fix for here in a moment. We're good to go!

**Pull request merged**